tests/e2e-openshift/export-to-cluster-logging-lokistack/otel-collector.yaml

```diff
@@ -0,0 +1,122 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+  namespace: openshift-logging
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector-logs-writer
+rules:
+ - apiGroups: ["loki.grafana.com"]
+   resourceNames: ["logs"]
+   resources: ["application"]
+   verbs: ["create", "get"]
+ - apiGroups: [""]
+   resources: ["pods", "namespaces", "nodes"]
+   verbs: ["get", "watch", "list"]
+ - apiGroups: ["apps"]
+   resources: ["replicasets"]
+   verbs: ["get", "list", "watch"]
+ - apiGroups: ["extensions"]
```

Comment:
Can we get rid of this? I don't think it's required on new clusters

Reply:
Ok, BTW When using k8s.deployment.name (which is enabled by default) or k8s.deployment.uid the processor needs get, watch and list permissions for replicasets resources. When using k8s.node.uid or extracting metadata from node, the processor needs get, watch and list permissions for nodes resources. In this case we are not using these attributes so I'll remove it.

Reply:
There are two `replicasets` one from `apps` group and another `extensions`. My understanding is that the extensions one is needed on old k8s cluster which we don't support anyways.